scripts/dependencies/tools.ts

```diff
@@ -229,6 +229,31 @@ export class GoLangCILint implements Dependency, GitHubDependency {
   }
 }
 
+export class CheckSpelling implements Dependency, GitHubDependency {
+  name = 'check-spelling';
+  githubOwner = 'check-spelling';
+  githubRepo = 'check-spelling';
+
+  download(context: DownloadContext): Promise<void> {
+    // We don't download anything there; the linting script does the cloning.
```

Comment:
It is unclear what "the listing script" is.

```suggestion
    // We don't download anything there; scripts/spelling.sh does the cloning.
```